help/destinations/catalog/advertising/Criteo.md

```diff
@@ -0,0 +1,143 @@
+
+
+## Prerequisites
+
+- You need to have an administrator user account on [Criteo Management Center](https://marketing.criteo.com)
+- You'll need your Criteo Advertiser ID (Ask your Criteo contact if you don't have this in hand)
+- Criteo only accepts SHA-256 and plain text emails (to be transformed into SHA-256 before sending). Please do not send any PII (Personal Identifiable Information such as individual's names or phone numbers)
+
+![Prerequisites](../../assets/catalog/advertising/criteo/criteo0.png)
+
+
+This is currently a beta product. Get in touch with your Criteo representative if you would like more details about what this means.
+
+## Supported identities
+
+_Criteo_ supports the activation of identities described in the table below. Learn more about [identities](https://experienceleague.adobe.com/docs/experience-platform/identity/namespaces.html?lang=en#getting-started).
+
+| Target Identity | Description | Considerations |
+| --- | --- | --- |
+| email\_sha256 | Email addresses hashed with the SHA-256 algorithm | Both plain text and SHA-256 hashed email addresses are supported by Adobe Experience Platform. When your source field contains unhashed attributes, check the  Apply transformation  option, to have Platform automatically hash the data on activation. |
+
+
+
+## Export type and frequency
+
+Refer to the table below for information about the destination export type and frequency.
+
+| Item | Type | Notes |
+| --- | --- | --- |
+| Export type | Profile-based | You are exporting all members of a segment, together with the desired schema fields (for example: email address, phone number, last name), as chosen in the select profile attributes screen of the [destination activation workflow](https://experienceleague.adobe.com/docs/experience-platform/destinations/ui/activate/activate-batch-profile-destinations.html?lang=en#select-attributes). |
+| Export frequency | Streaming | Streaming destinations are "always on" API-based connections. As soon as a profile is updated in Experience Platform based on segment evaluation, the connector sends the update downstream to the destination platform. Read more about [streaming destinations](https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-types.html?lang=en#streaming-destinations). |
+
+## Use cases
+
+To help you better understand how to use Criteo's destination, here are some goals that Adobe Experience Platform customers can achieve with Criteo.
+
+### Use case 1
+
+Get traffic
+
+Showcase your business with relevant product offers and flexible creatives. With intelligent product recommendations, your ads will automatically feature the products most likely to trigger visits and engagement. Flexible targeting allows you to build audiences from Criteo's commerce data set or from your own prospect lists and Adobe CDP segments
```

Comment:
Full stop at the end of the last phrase.

Reply:
Done.